pubspec.yaml

```diff
@@ -12,10 +12,14 @@ dependencies:
   json_pointer: ^0.1.0
   logging: '>=0.9.3 <2.0.0'
   path: ^1.3.0
-  uri: ">=0.11.1 <0.12.0"
+  uri: '>=0.11.1 <2.0.0'
   w_transport: ^4.0.0
 
 dev_dependencies:
+  build_runner: '>=1.7.2 <3.0.0'
+  build_test: ^0.10.10
+  build_web_compilers: ^2.9.0
+  build_vm_compilers: ^1.0.4
```

Comment:
fyi build_vm_compilers is effectively deprecated at this point, the `test` runner itself is faster at compiling dart tests for the VM so you can drop this dependency if you want.

Reply:
Ended up removing a few more, thanks for the suggestion!